crates/core/src/ring.rs

```diff
@@ -43,155 +40,19 @@ use crate::{
 
 mod connection_manager;
 pub(crate) use connection_manager::ConnectionManager;
+mod connection;
+pub mod live_tx;
```

Comment:
does this need to be `pub` ? can't we just re-export whatever type we are using outside of this module here?